File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SpiderRobe.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.LightArmor;

/* JADX INFO: loaded from: classes3.dex */
public class SpiderRobe extends LightArmor {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.armor_light_spider_robe_name;
        this.idDescription = R.string.armor_light_spider_robe_description;
        this.idImage = R.drawable.spider_robe;
        this.price = 540L;
        this.maxHp = 70;
        this.intelligence = 21;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SpiderRobe.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.LightArmor

class SpiderRobe : LightArmor() {
    override fun configureProperties() {
        idName = R.string.armor_light_spider_robe_name
        idDescription = R.string.armor_light_spider_robe_description
        idImage = R.drawable.spider_robe
        price = 540L
        maxHp = 70
        intelligence = 21
    }
}
